content/intel-me.md

```diff
@@ -0,0 +1,38 @@
+---
+title: What is the Intel Management Engine?
+description: >
+    This article describes the Intel Management Engine, scenarios in which the IME is disabled, and how to manually disable on systems running coreboot.
+keywords:
+  - Firmware
+  - Intel ME
+  - Management Engine
+  - Disable Intel ME
+  - IME
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+---
+
+System76 laptops and desktops with Open Firmware will typically have the Intel Management Engine (IME) disabled, unless doing so breaks functionality. Machines running Open Firmware with coreboot can disable the IME manually.
```

Comment:
I would be a bit softer on when we disable IME:

System76 laptop will have the Intel Management Engine (IME) disabled when it does not break functionality. System76 open firmware system do have some control to enable and disable the IME as [documented here](https://github.com/system76/firmware-open/blob/master/docs/intel-me.md)

Reply:
Implemented your suggested wording.